tests/jballerina-unit-test/src/test/resources/test-src/query/query_action_or_expr_semantic_negative.bal

```diff
@@ -318,3 +318,56 @@ function testQueryActionOrExprWithActionsInJoinClause() {
 }
 
 int[][] globalQuery = from var i in 1...3 select start f1();
+
+type T1 record {
```

Comment:
Can you add few ambiguity tests as well

Reply:
Added only single test since few of other tests are already found [here](https://github.com/ballerina-platform/ballerina-lang/blob/7453c9d785cddf54815b718464c8911c314c43ab/tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/query/QueryNegativeTests.java#L121)